tests/functional/spec/components/checkbox/checkbox_detail_answer_dropdown.spec.js

```diff
@@ -0,0 +1,91 @@
+import CheckboxDropdownPage from "../../../generated_pages/checkbox_detail_answer_dropdown/optional-checkbox-with-dropdown-detail-answer-block.page";
+import SummaryPage from "../../../generated_pages/checkbox_detail_answer_dropdown/summary.page";
+import DropdownMandatoryPage from "../../../generated_pages/dropdown_mandatory/dropdown-mandatory.page";
+
+describe("Optional Checkbox with a Dropdown detail answer", () => {
+  beforeEach(() => {
+    browser.openQuestionnaire("test_checkbox_detail_answer_dropdown.json");
+  });
+
+  describe("Given an optional checkbox with a dropdown detail answer", () => {
+    it("When a placeholder choice is provided for the detail answer dropdown, Then then provided placeholder should be displayed as the first option'", () => {
+      $(CheckboxDropdownPage.fruit()).click();
+
+      expect($(CheckboxDropdownPage.fruitDetail()).getText()).to.contain("Select fruit");
+    });
+
+    it("When a placeholder choice is not provided for the detail answer dropdown, Then the default placeholder should be displayed as the first option'", () => {
+      $(CheckboxDropdownPage.jam()).click();
+
+      expect($(CheckboxDropdownPage.jamDetail()).getText()).to.contain("Select an answer");
+    });
+
+    it("When the user does not provide an answer and submits, Then the summary should display 'No answer provided'", () => {
+      $(CheckboxDropdownPage.submit()).click();
+
+      expect($(SummaryPage.optionalCheckboxWithDropdownDetailAnswer()).getText()).to.equal("No answer provided");
+    });
+
+    it("When the user selects an option with an optional detail answer but does not provide a detail answer, Then the summary should display the chosen option without the detail answer", () => {
+      $(CheckboxDropdownPage.fruit()).click();
+      $(CheckboxDropdownPage.submit()).click();
+
+      expect($(SummaryPage.optionalCheckboxWithDropdownDetailAnswer()).getText()).to.equal("Fruit");
+    });
+
+    it("When the user selects an option with an optional detail answer and provides a detail answer, Then the summary should display the chosen option and its details", () => {
+      $(CheckboxDropdownPage.fruit()).click();
+      $(CheckboxDropdownPage.fruitDetail()).selectByAttribute("value", "Mango");
+      $(CheckboxDropdownPage.submit()).click();
+
+      expect($(SummaryPage.optionalCheckboxWithDropdownDetailAnswer()).getText()).to.equal("Fruit\nMango");
+    });
+
+    it("When the user selects the default dropdown option after submitting a detail answer, Then the summary should not display the detail answer", () => {
```

Comment:
This doesn't read like the others (i.e When the user selects an option with an optional detail answer)

Reply:
Not quite sure what you mean. What do you think it should be?

Reply:
When the user selects an option with an optional detail answer and doesn't change the value from the default ...

Reply:
The importance here is that they also already previously chose one, so what do you suggest fitting that phrase in?. In summary, the test is, select a non default detail answer, go back and select the default value, and the answer should be removed.